shared/src/main/scala/org/jetbrains/sbt/structure/dataSerializers.scala

```diff
@@ -138,14 +138,21 @@ trait DataSerializers {
         {what.home.toSeq.map { file =>
         <home>{file.path}</home>
       }}
-        {what.options.map { option =>
-        <option>{option.canonIfFile}</option>
+        {what.options.map { case (k, v) =>
```

Comment:
Serializer & Deserializer logic is duplicated in two classes.
Check if it's possible to extract the logic somehow. Maybe to some separate serializer.
It's better to do it once you extract options map to a separate class (see comment above)